Header header

geometry_msgs/Quaternion q
geometry_msgs/Vector3 r
geometry_msgs/Vector3 v
geometry_msgs/Vector3 Omega

float64 m
geometry_msgs/Vector3 I
geometry_msgs/Vector3 r_BM

geometry_msgs/Vector3 r_BP
geometry_msgs/Vector3 r_BI
geometry_msgs/Vector3 b_a
geometry_msgs/Vector3 b_Omega

float64[30] covariance
